Makefile

```
@@ -44,11 +44,13 @@ $(GENERATORBIN):
 $(GENERATORBIN)/generator: $(G_SRCS) | $(GENERATORBIN)
 	@crystal build $(GENERATORDIR)/generator.$(FILEEXT) -o generator/bin/generator
 
-build-generator: $(GENERATORBIN)/generator
+build-generator:
```

```suggestion
.PHONY: build-generator
build-generator:
```

On the off-chance that someone might actually put a `build-generator` file or executable here, or rename the target to something that matches a file, this would prevent Make thinking it should only run this target if the file modification time was updated.